Jonathan-jonathankern/Homework-1/99-bottles-of-beer/app.js

```diff
@@ -0,0 +1,31 @@
+/*
+  Write a script that prints the lyrics to "99 Bottles of Beer on the Wall" in the terminal.
+  If you're unfamiliar with the song, you can [find the lyrics here](http://www.99-bottles-of-beer.net/lyrics.html).
+
+  Be sure that all of your output includes proper spacing (no words or values running into each other).
+
+  Hint: You can make your output appear on multiple lines by using the string \n to insert a line break.
+
+  For an extra challenge, improve this code to print "1 bottle" rather than "1 bottles." (Hint: read up on if/else
+  conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else).
+*/
+
+'use strict';
+
+// if i is greater than 0 and less than 100, subtract 1
+for (let i = 99; i > 0 && i < 100; i--) {
```

Comment:
Your counter starts at 99 and decrements, meaning it's never going to be greater than 99, so there's no need to check whether it's less than 100 here. This doesn't affect the execution of your code, but when your code can be simpler and achieve the same results, that's always better!